spec/support/shared_examples_for_personal_keys.rb

```diff
@@ -36,6 +38,13 @@
       copied_text = page.evaluate_async_script('navigator.clipboard.readText().then(arguments[0])')
 
       expect(copied_text).to eq(scrape_personal_key)
+
+      click_continue
+      page.find(':focus').send_keys [mac? ? :meta : :control, 'v']
```

Comment:
maybe some parens here for clarity?
```suggestion
      page.find(':focus').send_keys([(mac? ? :meta : :control), 'v'])
```

Reply:
I dunno that I personally find that to be much clearer. If the current implementation is a bit hard to parse, I might sooner opt for splitting out a variable for the modifier? WDYT?

```suggestion
      mod = mac? ? :meta : :control
      page.find(':focus').send_keys [mod, 'v']
```

Reply:
yeah definitely like the separate variable approach

Reply:
👍 Updated in 56d479636feb9440a38dbc572f39cb7287c04db2